documentation/source/overview/studies.rst

```diff
@@ -5,96 +5,96 @@ Studies using SCT
 
 The following studies (in chronological order) have used SCT:
```

Comment:
While we're at it, here's one other minor formatting suggestion: What do you think about adding headers for each year?

Reply:
Thanks for the feedback, @joshuacwnewton but didn't get you. Something like:
2021:
- Paper 1
- Paper 2

2020:
- Paper 1 
- Paper 2

etc etc.?

Reply:
Exactly!

Reply:
All papers are in the respective years as a list now :)